Modules/SubModule_Import/xASL_imp_NII2BIDS_Session.m

```diff
@@ -30,6 +30,10 @@
     %% Unpack bidsLabel struct
     subjectLabel = bidsLabel.subject;
```

Comment:
Why do we "unpack"? This only means that we have twice the number of temporary variables right?
Can't we simply use `bidsLabel.subject` throughout? Same for the visit one.

Reply:
Yeah, this is there because we were working on reducing the number of io arguments while keeping the code stable. This was a nice temporary solution. Of course we can remove the "unpacking" and just use bidsLabel throughout the code. We just haven't had time to do this before and there was a time when the code was super fragile 😄

Reply:
Fixed.